### ISSUE TITLE
AutoHide does not work

### ISSUE THREAD

**Issue by idanyadgar** [MurhafSousli/ngx-scrollbar#76]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### OS and Version?
> Windows 10

### Versions
Angular CLI: 7.0.4
Node: 10.8.0
OS: win32 x64
Angular: 7.0.2

### Repro steps
The README has a stackblitz link: https://stackblitz.com/edit/ngx-scrollbar
Setting autoHide there does not work either.

**Comment by MurhafSousli** [comment]
That's weird, I tested it before publishing and it was working, current workaround

```scss
::ng-deep {
  .ng-scrollbar-auto-hide .ng-scrollbar {
    opacity: 0;
    transition: opacity 120ms ease-out;
  }
  .ng-scrollbar-auto-hide:hover .ng-scrollbar,
  .ng-scrollbar-auto-hide:active .ng-scrollbar,
  .ng-scrollbar-auto-hide:focus .ng-scrollbar {
    opacity: 1;
    transition: opacity 340ms ease-out;
  }
}
```

**Comment by MurhafSousli** [comment]
fixed in 3.1.1
